website/www/site/content/en/documentation/sdks/python-streaming.md

```diff
@@ -168,9 +168,7 @@ Python streaming execution does not currently support the following features.
 
 These unsupported Beam features apply to all runners.
 
-- `State` and `Timers` APIs
 - Custom source API
-- Splittable `DoFn` API
 - Handling of late data
```

Comment:
I believe handling of late data is also supported now. @angoenka / @y1chi could you confirm?

Reply:
Yes, late data is also supported.

Reply:
Thanks, I removed the entire unsupported features section.

Reply:
The only real remaining item, "User-defined custom merging `WindowFn` (with fnapi)" is specific to portability, will add a pointer to the portable runner support table instead.